docs/client-configuration.md

```diff
@@ -28,13 +28,30 @@
 
 ## Reconnect Strategy
 
-You can implement a custom reconnect strategy as a function:
+When a network error occurs the client will automatically try to reconnect, following a default linear strategy (the more attempts, the more waiting before trying to reconnect).
+
+This strategy can be overriden by providing a `socket.reconnectStrategy` option during client's creation.
```

Comment:
1. overridden
2. during **the** client's creation.

Reply:
fixed